Eatery Blue/Eatery Blue/UI/EateryScreen/EateryViewController.swift

```diff
@@ -287,6 +287,17 @@ class EateryViewController: UIViewController {
         }
     }
 
+    func addAlertInfoView(_ message: String) {
```

Comment:
Same here. What's the reasoning behind `addAlertInfoView` instead of `setUpAlertInfoView`